tests/test_specific_versions.py

```diff
@@ -51,6 +51,25 @@ def test_simple_dump_and_load_dataclass_with_future_import(self):
 
         self.assertDictEqual(expected, dumped)
 
+    @only_version_3(7, and_above=False)
+    def test_postponed_annoation_dataclass(self):
+        from postposned_dataclass import A
```

Comment:
There's a typo in the module name that's leaked into the code here.

```suggestion
        from postponed_dataclass import A
```